src/main/kotlin/org/malachite/estella/DemoApplication.kt

```diff
@@ -27,24 +28,51 @@ class DemoApplication {
         @Autowired userRepository: HibernateUserRepository,
         @Autowired offerRepository: HibernateOfferRepository,
         @Autowired recruitmentProcessRepository: HibernateRecruitmentProcessRepository,
-        @Autowired desiredSkillRepository: HibernateDesiredSkillRepository
+        @Autowired desiredSkillRepository: HibernateDesiredSkillRepository,
+        @Value("\${should_fake_load}") shouldFakeLoad: Boolean
```

Comment:
Ta wartość pokazuje, czy jest to test, czy nie i czy powinien fake loader się na początku odpalić. Przy testach tego nie chcemy, bo potem się dość dużo psuje

Reply:
Czyli teraz testy będą się odpalać na pustej bazie?

Reply:
Nie, ale nie opiszę tu tego, wytłumaczę Ci na głosowym XD